tests/core/tests/test_widgets.py

```diff
@@ -217,7 +217,7 @@ class ManyToManyWidget(TestCase):
     def setUp(self):
         self.widget = widgets.ManyToManyWidget(Category)
         self.widget_name = widgets.ManyToManyWidget(Category, field="name")
-        self.cat1 = Category.objects.create(name='Cat úňíčóďě')
+        self.cat1 = Category.objects.create(name='Cat úňíčóďě'.encode())
```

Comment:
There is no need for `encode` here

Reply:
Yeah I thought that too. However, after reading and experimenting a bit, it seems that is the only way to get the tests passing. 😟 

Ill try what you suggested above and see if it works.

Reply:
It look like it is still failing. I'll take a look at it as soon as I catch a little time today

Reply:
No worries. The encoding seems to make it work. So either its a setting issue in MySQL which needs configuring, or less likely, stings need `encode` explicitly.

Reply:
And FYI, you can run it locally with ` docker run -e MYSQL_ROOT_PASSWORD=password -d -p 3306:3306 mysql` and then `IMPORT_EXPORT_TEST_TYPE=mysql-innodb ./runtests.sh`

Reply:
ok, I see that test database had default collation `latin1_swedish_ci`. 

I have added commit and squased in:
https://github.com/django-import-export/django-import-export/pull/905

Reply:
`latin1_swedish_ci`? 😕 

Any idea why? Is that a travis thing?

Reply:
no, `latin1_swedish_ci`  is default :)

Reply:
Clearly this makes sense only because I'm swedish! 😆